src/app/components/Block/media.scss

```diff
@@ -29,6 +29,46 @@
     }
   }
 
+  .phone-frame {
+    height: 100%;
+    max-height: 85vh;
+
+    filter: drop-shadow(2px 4px 6px white);
```

Comment:
I think this is kind of ugly, and I don't think we use drop-shadows anywhere else in Odyssey.

Do we need it as a default or could we have a plain video with no border?

Reply:
Yep, good call. I'll move this and the border radius over to the tiktok JS